torch/testing/_internal/common_utils.py

```diff
@@ -1770,10 +1773,17 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, hig
         result = torch.repeat_interleave(result, 2, dim=-1)
         result = result[..., ::2]
 
+    if not include_zero and dtype not in complex_types():
+        if dtype in integral_types() or dtype is torch.bool:
+            replace_with = torch.tensor(1, device=device, dtype=dtype)
+        elif dtype in floating_types_and(torch.half, torch.bfloat16):
+            replace_with = torch.tensor(sys.float_info.epsilon, device=device, dtype=dtype)
```

Comment:
use torch.finfo for epsilon: https://pytorch.org/docs/stable/type_info.html

Reply:
Yes, thanks @kshitij12345. I'm sorry, I was working on the docs, will fix this along with that commit.